org-code-javabuilder/media/src/main/java/org/code/media/support/MediaRuntimeExceptionKeys.java

```diff
@@ -1,5 +1,6 @@
 package org.code.media.support;
 
 public enum MediaRuntimeExceptionKeys {
-  IMAGE_LOAD_ERROR
+  IMAGE_LOAD_ERROR,
+  FILE_NOT_FOUND
```

Comment:
Do we need to translate these in javalab, or do we already handle this?

Reply:
Nope thankfully! The actual value (`"FILE_NOT_FOUND"`) is already mapped.

Reply:
I guess generally I'm curious about how we manage when message keys are used in multiple places in Javabuilder -- is it just on us to make sure that if we use the same key in multiple places that the message that we'd want to display on the client is the same (ie, "many to one")?

Reply:
Yeah that's a good question - I think this is one of the only (if not the only) example where a key is defined in multiple enums. In this case it was mostly a matter of convenience since the translation already existed, but yeah I do wonder if we need some more clarity around how we define these.